docs/install/containers/recipes/index.md

```diff
@@ -118,3 +118,41 @@ RUN <<EOT
 
 After making these changes, build the project container as usual.
 It will no longer output the access log, but will continue to output the event log.
+
+
+## Pack the ZODB
+
+A common maintenance task of a Plone instance is to pack the ZODB.
+
+The official `plone/plone-backend` container and project containers based on them have a `pack` command to pack the ZODB.
```

Comment:
If this is moved into https://6.docs.plone.org/install/containers/images/backend.html, then ignore this suggestion.

```suggestion
The official {doc}`/install/containers/images/backend` container and project containers based on them have a `pack` command to pack the ZODB.
```